src/pages/public_registration/LogPanel.tsx

```diff
@@ -13,6 +13,21 @@ import { CompletedPublishingRequestStatusBox } from './action_accordions/Complet
 const ticketStatusesToShow: TicketStatus[] = ['Completed', 'Closed'];
 const ticketTypesToShow: TicketType[] = ['PublishingRequest', 'DoiRequest'];
 
+type LogList = (Ticket | NonTicketLogEntry)[];
+
+interface NonTicketLogEntry {
+  modifiedDate: string;
+  description: string;
+}
+
+function isTicket(object: any): object is Ticket {
+  return ('type' as TicketType) in object;
+}
+
+function isNonTicketLogEntry(object: any): object is NonTicketLogEntry {
```

Comment:
`object is Ticket` var snedig, har ikke brukt den før. I like it! 🤩 
Ikke noe i veien for å oppgi den originale typen til `object` heller? Om du syns det blir i overkant bloated er det mulig å legge til en `type LogItem = Ticket | NonTicketLogEntry` som brukes i stedet også

```suggestion
function isTicket(object: Ticket | NonTicketLogEntry): object is Ticket {
  return ('type' as TicketType) in object;
}

function isNonTicketLogEntry(object: Ticket | NonTicketLogEntry): object is NonTicketLogEntry {
```